### ISSUE TITLE
Error "all_data["graphql"]["location"]["edge_location_to_media"] KeyError: 'graphql'"

### ISSUE THREAD

**Issue by PepiMarquez** [instabot-py/instabot.py#2216]
Hello,
im getting this error, somebody can help me??

Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [x] If so, have you tried running the code on your local machine?

#### Operating System? (include version)
- [ ] macOS
- [x] Windows
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
Include here the response of 'python --version' or 'python3 --version'
```

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
```

#### Description of your issue

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

#### Config

```
Include the contents of your config here (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT
```

**Comment by PepiMarquez** [comment]
Im getting this error: Error "all_data["graphql"]["location"]["edge_location_to_media"] KeyError: 'graphql'" 
I try all the issues was here...but i still have the problem.

**Comment by nucleartux** [comment]
I have same problem with latest stable version:
```
2019-06-27 04:42:58,671 - InstaBot - ERROR - get_media_id_by_tag
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/instabot_py/instabot.py", line 528, in get_media_id_by_tag
    all_data["graphql"]["hashtag"]["edge_hashtag_to_media"][
KeyError: 'graphql'
```

**Comment by mrallanodreman** [comment]
im having the same issue but i feel that is not afecting the perfomance! 
im having only Location target and im working fine on Android
but i see the error 

"2019-06-27 04:42:58,671 - InstaBot - ERROR - get_media_id_by_tag
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/instabot_py/instabot.py", line 528, in get_media_id_by_tag
    all_data["graphql"]["hashtag"]["edge_hashtag_to_media"][
KeyError: 'graphql'" 

frecuently on my log....

**Comment by mrallanodreman** [comment]
FIXED

one of the target was a l:xxxxxx and in the input it was l:spacexxxxxx so is a type problema!

**Comment by yurilaaziz** [comment]
fixed in the 0.5.2 release https://medium.com/@asker_amine/whats-new-in-the-instagram-bot-0-5-2-fe52154fdc8a
